float64 xPos
float64 yPos
float64[] rotationQuaternion
float64 micHeight
float64 micAngle
bool isFlashing
float64 ledR
float64 ledG
float64 ledB
float64 flashFrequency